templates/front/src/index.tsx

```diff
@@ -30,7 +31,9 @@ initializeLocalizedDate(injector.getInstance(LocaleService));
 
 const reactApp: JSX.Element = (
   <React.StrictMode>
-    <App />
+    <RouteProvider>
```

Comment:
pour moi le route provider doit etre dans App, l'index expose juste l'App

Reply:
Il faut juste voir comment ça se passe en SSR, parfois c'est plus intéressant de sortir de router provider pour être capable dans un environnement SSR de faire le rendu du app dans un contexte de routes différent (in memory vs browser based)

Reply:
Alright je comprends.

J'ai quand même un peu de mal avec le découpage et la déclaration des routes. Je pense qu'il faudrait crashtester sur l'admin pour voir si on a pas de soucis (sorry c'est un peu à l'encontre du but de cette MR)

type-route a une API pour le SSR ? je trouve pas sur le net

Reply:
https://type-route.zilch.dev/guides/server-side-rendering.html

Reply:
Selon ton exemple, le RouterProvider change pas entre un contexte client ou SSR, ca à l'air de passer par l'objet session exporté par type-route